apps/pocketbase/main.go

```diff
@@ -134,22 +166,55 @@ func main() {
 
 				// submit the form & create the record
 				err = form.Submit()
-
-				// if the submition failed, find if there is an existing record with the same hash
 				if err != nil {
-					// if the hash exists, return the existing record
-					existingRecord, err := app.Dao().FindFirstRecordByData("maps", "hash", hashEncoded)
-					if err != nil {
-						return err
-					}
-					if existingRecord != nil {
-						return c.JSON(http.StatusOK, MapRouteReturn{
-							Id: existingRecord.Id,
-						})
-					}
 					return err
 				}
 
+				// create a new browserless context
+				allocatorContext, cancel := chromedp.NewRemoteAllocator(context.Background(), *browserlessConnection)
+				defer cancel()
+				ctx, cancel := chromedp.NewContext(allocatorContext)
+				defer cancel()
+
+				// create a new browserless task
+				var screenshot []byte
+				if err := chromedp.Run(ctx,
+					chromedp.EmulateViewport(1600, 900),
+					chromedp.Navigate(browserlessFrontendURI + "/app?m=" + record.Id),
+					chromedp.WaitReady("#testing-map"),
+					chromedp.FullScreenshot(&screenshot, 100),
```

Comment:
I know you said that this would be broken apart into another issue, but I found an example of a single element screenshot in [chromedp examples](https://github.com/chromedp/examples/blob/master/screenshot/main.go).

`elementScreenshot(`https://pkg.go.dev/`, `img.Homepage-logo`, &buf)`

```
func elementScreenshot(urlstr, sel string, res *[]byte) chromedp.Tasks {
	return chromedp.Tasks{
		chromedp.Navigate(urlstr),
		chromedp.Screenshot(sel, res, chromedp.NodeVisible),
	}
}
```

Reply:
Yeah, but then we can't control the size of the screenshot easily. Because the sidebar and topbar use up a certain amount of pixels. I'd rather add something to the frontend that renders the map as it should be screenshot.